---
title: Analyses of brain gene expression and DNA methylation in Alzheimer’s disease model mice - effects of perinatal choline nutrition
pi: Jan Krzysztof Blusztajn PhD, Department of Pathology and Laboratory Medicine
intern: Navin Ramanan
year: 2022
---

Alzheimer’s Disease (AD) is a progressive neurodegenerative disease and the most common form of dementia. Recent
estimates suggest that almost 6 million people suffer from AD in the US. Although there has been a keen interest in
finding treatments for AD, there is a tremendous lack in disease-altering treatment options. Thus, prevention of AD has
become an interesting topic. Supplementation with the essential nutrient, choline, has been found to improve cholinergic
functioning, enhance long-term potentiation, and improve spatial memory in rodents, and also reduce amyloid beta
pathology in mouse models of AD. Additionally, choline is a methyl donor group and altering early-life methyl donor
group levels has been shown to alter DNA methylation. Interestingly, changes in DNA methylation levels naturally occur
with aging. These DNA methylation changes appear in a typified pattern that lead to what is known as the “methylation
clock” that enables the identification of the methylation – or biological – age of a sample. Furthermore, people with AD
show an acceleration of these DNA methylation changes and thus methylation age. Since we know that choline can act as a
methyl donor group and choline supplementation protects against age-related cognitive decline and attenuates AD-like
pathology in animal models, we hypothesize that these beneficial outcomes of choline supplementation may be mediated
through changes in DNA methylation and altered patterns of brain gene expression. In order to elucidate this, we have
gathered bulk RNA sequencing and DNA methylation data from several brain regions of wildtype and AD-model mice across 4
timepoints (3, 6, 9, and 12 months of age) with either choline-supplemented or control perinatal diets. RNA-sequencing
has produced a robust dataset of 192 FASTQ files that will be analyzed using standard bioinformatic analysis to trim,
align, normalize, count, and identify differentially expressed genes across age, diet, genotype, sex, and brain region.
Pathway analysis and gene set enrichment analysis (GSEA) will also be conducted using these differentially expressed
gene sets to identify biologically relevant systems that are altered. Furthermore, DNA methylation analyses will be
performed on the same samples using reduced representation bisulfite sequencing and differentially methylated CpGs and
regions will be identified using similar methods to those for differentially-expressed transcripts and methylation age
will be generated for each mouse using methods of Stubbs et al., 2017 (PMID 28399939). Finally, correlations between
differentially methylated DNA patterns and gene expression will be identified to elucidate any possible cis-acting
regulatory sites in relation to age, diet, genotype, sex and brain region.    
